src/components/common/form/OtpInput.js

```diff
@@ -84,11 +86,19 @@ const Input = ({ min, max, pattern, focus, shouldAutoFocus, onChange, ...props }
   useEffect(() => {
     if (input && focus) {
       input.focus()
+      setSelection({ start: 0, end: value && value.length ? 1 : 0 })
```

Comment:
why not use value.length instead of 1?

Reply:
it's the same, each input should have between 0 and 1 characters, not more...